lib/gui/pages/main/styles/_main.scss

```diff
@@ -106,3 +106,7 @@
 .page-main .step-size {
   color: $palette-theme-dark-disabled-foreground;
 }
+
+.page-main .popover-content {
+  -webkit-user-select: auto;
```

Comment:
I think if the user wants to see the full path of the image, they may want to copy that path wholly or partially too, so allowing selections here. I assumed it was the case for the modal to match its features, but I just now checked that it isn't, so this isn't necessary but could be nice.